src/components/Tile/Tile.module.scss

```diff
@@ -1,11 +1,11 @@
 @use '@assets/global.scss' as g;
 
 .tile {
-  background: g.$primary-white;
+  background: white;
   padding: 40px;
   border-radius: 10px;
   flex: 0 0 100%;
-  border: 0.5px solid grey;
+  border: 0.5px solid #e5e6eb;
```

Comment:
Please use the `$greyscale-disabled` variable from the globals file (and also apply to the line below too, I know that wasn't you, but while you're there)